librz/arch/function.c

```diff
@@ -258,7 +258,22 @@ RZ_API bool rz_analysis_function_relocate(RzAnalysisFunction *fcn, ut64 addr) {
 	return true;
 }
 
-RZ_API bool rz_analysis_function_rename(RzAnalysisFunction *fcn, const char *name) {
+/**
+ * @b Rename given function.
```

Comment:
Use `\p`, etc Doxygen syntax

Reply:
Isn't this doxygen syntax as well?
> Structural commands (like [all other commands](https://doxygen.nl/manual/commands.html#cmd_intro)) start with a backslash (\), or an at-sign (@) if you prefer Javadoc style, followed by a command name and one or more parameters.

Reply:
Yes, but we use the `\p` one

Reply:
👍 sorry for not checking that 😅 , I'll make the change

Reply:
I have a habit of using `@` everywhere :-)